docs/src/index.md

```diff
@@ -118,16 +118,30 @@ skate('my-component', {
   // - oldValue: The old value. If type === 'insert', this will be undefined.
   attributes: {
     'my-attribute': {
+      // The element's default value if not already specified on the element.
+      // Can also be a function that returns a value.
+      default: 'default value',
+
+      // Called when the attribute is set for the first time.
       created: function (element, change) {
 
       },
 
+      // Called on all attribute modifications except for the intial creation.
       updated: function (element, change) {
 
       },
 
+      // Called when the attribute is removed.
       removed: function (element, change) {
 
+      },
+
+      // Called when created, updated or remove is not specified. This is the
+      // same as specifying a function instead of an options object for a
+      // given attribute.
+      catchall: function (element, change) {
```

Comment:
>  I don't like the name 'catchall', maybe something like 'always'?

The idea of this callback and it's current name are a bit confusing.

I think it'd be simpler to have an `always` callback ([like in jQuery](http://api.jquery.com/deferred.always/)) and just always call it, even if you already have a callback for that specific lifecycle event. You'd need to add an argument that specifies the type, though.